src/main/java/pl/nogacz/snake/board/Board.java

```diff
@@ -116,6 +174,98 @@ private void moveSnakeHead(Coordinates coordinates) {
         }
     }
 
+    private void spawnRottenApple(){
+        if(Math.abs(lastSpawnTime - System.currentTimeMillis())/1000 == randSpawnRottenAppleTime){
+            addRottenApples();
+            rottenApples.add(rottenAppleCurrentCoordinates);
+            lastSpawnTime = System.currentTimeMillis();
+            randSpawnRottenAppleTime = random.nextInt(9)+1;
+            System.out.println("random spawn time:" + randSpawnRottenAppleTime);
+        }
+
+        if(Math.abs(lastSpawnTime2 - System.currentTimeMillis())/1000 == randSpawnRottenAppleTime2){
+            addRottenApples2();
+            rottenApples2.add(rottenAppleCurrentCoordinates2);
+            lastSpawnTime2 = System.currentTimeMillis();
+            randSpawnRottenAppleTime2 = random.nextInt(20)+1;
+            System.out.println("random spawn time2:" + randSpawnRottenAppleTime2);
+        }
+
+        if(Math.abs(lastSpawnTime3 - System.currentTimeMillis())/1000 == randSpawnRottenAppleTime3){
+            addRottenApples3();
+            rottenApples3.add(rottenAppleCurrentCoordinates3);
+            lastSpawnTime3 = System.currentTimeMillis();
+            randSpawnRottenAppleTime3 = random.nextInt(15)+1;
+            System.out.println("random spawn time3:" + randSpawnRottenAppleTime3);

```

Comment:
Debugging için koyduğun sysout satırlarını koddan çıkar.

Reply:
done.